src/mailadm/cmdline.py

```diff
@@ -179,80 +177,32 @@ def gen_qr(ctx, tokenname):
               default="https://example.org/new_email", show_default="https://example.org/new_email")
 @click.option("--mail-domain", type=str, prompt="mail domain for which we create new users",
               default="example.org", show_default="example.org")
-@click.option("--vmail-user", type=str, default="vmail",
-              help="dovecot virtual mail delivery user")
-@click.option("--path-virtual-mailboxes", default=None, type=str,
-              help="postfix virtual users map, generated by mailadm")
+@click.option("--mailcow-endpoint", type=str, default=None,
+              help="the API endpoint of the mailcow instance")
+@click.option("--mailcow-token", type=str, default=None,
+              help="you can get an API token in the mailcow web interface")
 @click.pass_context
-def init(ctx, web_endpoint, mail_domain, vmail_user, path_virtual_mailboxes):
+def init(ctx, web_endpoint, mail_domain, mailcow_endpoint, mailcow_token):
     """(re-)initialize configuration in mailadm database.
 
     Warnings: init can be called multiple times but if you are doing this to a
     database that already has users and tokens, you might run into trouble,
     depending on what you changed.
     """
+    if not mailcow_endpoint:
+        mailcow_endpoint = "https://%s/api/v1/" % (mail_domain,)
+
```

Comment:
i wouldn't do this automatically but rather make the arguments required.

Reply:
makes sense.